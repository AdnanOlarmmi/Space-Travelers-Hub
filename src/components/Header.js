import React from 'react';
import { Link } from 'react-router-dom';

const Header = () => (
  <header className="flex-row">
    <div className="flex-row-center">
      <img className="icon" src="https://cdn-icons-png.flaticon.com/512/3212/3212608.png" alt="Space Travelers Hub Logo" />
      <h5>
        Space Travelers&apos; Hub
      </h5>
    </div>
    <nav>
      <ul className="nav flex-row">
        <Link to='/'>
        <li className='nav__rockets'>Rockets</li>
        </Link>
        <Link to='/missions'>
        <li>Missions</li>
        </Link>
        <Link to='/myprofile'>
        <li className='nav__profile'>My Profile</li>
        </Link>  
      </ul>
    </nav>
  </header>
);

export default Header;
