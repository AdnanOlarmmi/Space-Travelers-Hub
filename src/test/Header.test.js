import Header from '../components/Header';

describe('Header', () => {
  it('render Header Components', () => {
    expect(Header).toMatchSnapshot();
  });
});
