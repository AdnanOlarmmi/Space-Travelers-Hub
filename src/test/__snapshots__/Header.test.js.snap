// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header render Header Components 1`] = `[Function]`;

exports[`MyRocket render My Rockets Components 1`] = `[Function]`;

exports[`Rockets Check if button works 1`] = `[Function]`;

exports[`Rockets render My Rockets Components 1`] = `[Function]`;
